pages/data-providers/index.jsx

```diff
@@ -49,9 +50,16 @@ const useDataProviderController = () => {
 const getHelperMessage = ({ created, duplicated, error }) => {
   if (error) {
     return {
-      message:
-        'We could not find any OAI-PMH endpoint.' +
-        ' Please try it with another URL.',
+      message: (
+        <>
+          We could not find any OAI-PMH endpoint. Please try it with another
+          URL. If you believe the URL is correct please contant us at{' '}
```

Comment:
```suggestion
          We could detect neither repository nor journal. Please, provide
          the exact OAI-PMH endpoint URL. If you are having trouble contact us at{' '}
```

Ideally, the 'contact us' should be a link to some other page or to a prefilled email request. But now it's fine as it is. Please, agree all the messages with @nancypontika or I will if it would be easier for you.